maven-xml/src/test/java/org/apache/maven/xml/filter/ConsumerPomXMLFilterTest.java

```diff
@@ -0,0 +1,168 @@
+            {
+                return Optional.empty();
+            }
+            
+            @Override
+            protected Optional<String> getChangelist()
+            {
+                return Optional.of( "CL" );
+            }
+            
+            @Override
+            protected Function<Path, Optional<RelativeProject>> getRelativePathMapper()
+            {
+                return null;
+            }
+            
+            @Override
+            protected Function<DependencyKey, String> getDependencyKeyToVersionMapper()
+            {
+                return null;
+            }
+        };
+        
+        Provider<BuildPomXMLFilterFactory> provider = new Provider<BuildPomXMLFilterFactory>()
+        {
+
+            @Override
+            public BuildPomXMLFilterFactory get()
+            {
+                return buildPomXMLFilterFactory;
+            }
+        };
+        
+        XMLFilter filter = new ConsumerPomXMLFilterFactory( provider )
+        {
+        }.get( Paths.get( "pom.xml" ) );
+        filter.setFeature( "http://xml.org/sax/features/namespaces", true );
+        return filter;
+    }
+    
+    @Test
+    public void testAllFilters() throws Exception {
+        String input = "<project>\n"
+                     + "  <parent>\n"
+                     + "    <groupId>GROUPID</groupId>\n"
+                     + "    <artifactId>PARENT</artifactId>\n"
+                     + "    <version>VERSION</version>\n"
+                     + "    <relativePath>../pom.xml</relativePath>\n"
+                     + "  </parent>\n"
+                     + "  <artifactId>PROJECT</artifactId>\n"
+                     + "  <modules>\n"
+                     + "    <module>ab</module>\n"
+                     + "    <module>../cd</module>\n"
+                     + "  </modules>\n"
+                     + "</project>";
+        String expected = "<project>\n"
+                        + "  <parent>\n"
+                        + "    <groupId>GROUPID</groupId>\n"
+                        + "    <artifactId>PARENT</artifactId>\n"
+                        + "    <version>VERSION</version>\n"
+                        + "    <relativePath/>\n"
+                        + "  </parent>\n"
+                        + "  <artifactId>PROJECT</artifactId>\n"
+                        + "</project>";
+        String actual = transform( input );
+        assertThat( actual ).and( expected ).ignoreWhitespace().areIdentical();
+    }
+    
+    @Test
+    public void testMe() throws Exception {
+        String input = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n" + 
+            "<project xmlns=\"http://maven.apache.org/POM/4.0.0\" \r\n" + 
+            "         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" + 
+            "         xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 \r\n" + 
+            "                             http://maven.apache.org/maven-v4_0_0.xsd\">\r\n" + 
```

Comment:
That's the old location.

Reply:
A test, but it is fine to update

Reply:
I've created https://issues.apache.org/jira/browse/MNG-6778 to fix that in the official poms.